src/clib/Make.mach.win10

```diff
@@ -0,0 +1,87 @@
+#=======================================================================
+#
+# FILE:        Make.mach.linux-gnu
+#
+# DESCRIPTION: Makefile settings for a machine running Ubuntu
+#
+# AUTHOR:      Rick Wagner (rick@ucsd.edu)
+#
+# DATE:        2008-09-16
+#
+# This configuration assumes that build-essentials, gfortran, 
+# OpenMPI and HDF5 have been installed using apt-get.
```

Comment:
This definitely needs updating! (Probably more than any other Makefile, the header here is confusing. Presumably this works when using windows subsystem for linux?)